src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -204,9 +209,13 @@ class AdoConnector {
 
             if (document.Any()) {
                 _ = await _client.UpdateWorkItemAsync(document, _project, (int)(item.Id!));
-                _logTracer.Info($"notify ado: updated {item.Id:Tag:WorkItemId} - {notificationInfo}");
+                var adoEventType = "AdoUpdate";
+                _logTracer.Event($"{adoEventType:Tag:AdoEventType} {item.Id:Tag:WorkItemId} {notificationInfo["notification_id"]:Tag:NotificationId} {notificationInfo["job_id"]:Tag:JobId} {notificationInfo["task_id"]:Tag:TaskId} {notificationInfo["ado_project"]:Tag:AdoProject} {notificationInfo["ado_url"]:Tag:AdoUrl} {notificationInfo["container"]:Tag:Container} {notificationInfo["filename"]:Tag:filename}");
+
             } else {
-                _logTracer.Info($"notify ado: no update {item.Id:Tag:WorkItemId} - {notificationInfo}");
+                var adoEventType = "AdoNoUpdate";
```

Comment:
Help me understand. I take it if we get here we have expected to find a work item, but we didn't. Doesn't that mean we should create one?

Reply:
That's not what this condition represents. This condition represents finding an existing work item, but OneFuzz has no additional changes to make to that work item. 

We create new work items elsewhere. It happens when no work item is found.